tcp/spring-cloud-starter-stream-processor-tcp-client/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-starter-stream-processor-tcp-client</artifactId>
+	<packaging>jar</packaging>
+	<name>spring-cloud-starter-stream-processor-tcp-client</name>
+	<description>Spring Cloud Stream tcp client processor</description>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<artifactId>spring-cloud-stream-tcp-parent</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-ip</artifactId>
```

Comment:
I don't see reason to copy/paste this dep all the time, since we can just remove that `<optional>true</optional>` from the `app-starters-tcp-common` and be good everywhere we need `IP` stuff.
The `spring-integration-ip` is so lightweight:
```
project('spring-integration-ip') {
	description = 'Spring Integration IP Support'
	dependencies {
		compile project(":spring-integration-core")
		testCompile project(":spring-integration-stream")
	}
}
```
 that it even can be default dep from the top level.
But that is fully different story...